english/forest/shelter/play-dead/get-out/get-out.md

```diff
@@ -0,0 +1,14 @@
+After leaving the improvised tent, you realize that it's been raining for a while now. The footprints of the bear are clearly visible in the wet terrain.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.